packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -18,8 +22,72 @@ export interface TabProps extends Omit<React.HTMLProps<HTMLAnchorElement | HTMLB
   tabContentRef?: React.RefObject<any>;
   /** whether to render the tab or not */
   isHidden?: boolean;
+  /** Adds disabled styling and disables the button using the disabled html attribute */
+  isDisabled?: boolean;
+  /** Adds disabled styling and communicates that the button is disabled using the aria-disabled html attribute */
+  isAriaDisabled?: boolean;
+  /** Events to prevent when the button is in an aria-disabled state */
+  inoperableEvents?: string[];
+  /** Forwarded ref */
+  innerRef?: React.Ref<any>;
 }
 
-/** The parent <Tabs> component accecesses this component's propeties directly in order to present each Tab */
-export const Tab: React.FunctionComponent<TabProps> = (_props: TabProps) => null;
+const TabBase: React.FunctionComponent<TabProps> = ({
+  title,
+  eventKey,
+  tabContentRef,
+  id: childId,
+  tabContentId,
+  className: childClassName = '',
+  ouiaId: childOuiaId,
+  isDisabled,
+  isAriaDisabled,
+  inoperableEvents = ['onClick', 'onKeyPress'],
+  href,
+  innerRef,
+  ...props
+}: TabProps) => {
+  const preventedEvents = inoperableEvents.reduce(
+    (handlers, eventToPrevent) => ({
+      ...handlers,
+      [eventToPrevent]: (event: React.SyntheticEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+      }
+    }),
+    {}
+  );
+  const { mountOnEnter, localActiveKey, unmountOnExit, uniqueId, handleTabClick } = React.useContext(TabsContext);
+  let ariaControls = tabContentId ? `${tabContentId}` : `pf-tab-section-${eventKey}-${childId || uniqueId}`;
+  if ((mountOnEnter || unmountOnExit) && eventKey !== localActiveKey) {
+    ariaControls = undefined;
+  }
+  return (
+    <li
+      className={css(styles.tabsItem, eventKey === localActiveKey && styles.modifiers.current, childClassName)}
+      ref={innerRef}
+    >
+      <TabButton
+        className={css(
+          styles.tabsLink,
+          isDisabled && href && styles.modifiers.disabled,
+          isAriaDisabled && styles.modifiers.ariaDisabled
+        )}
+        disabled={isDisabled}
+        aria-disabled={isDisabled || isAriaDisabled}
```

Comment:
It doesn't hurt anything, but you don't need `aria-disabled="true"` when it's a `button` with the `disabled` attribute.

Reply:
Thanks for pointing that out, fixed it and added disabled state to nav examples